src/Tabs/Tabs.js

```diff
@@ -113,6 +114,29 @@ class Tabs extends Component {
     this.setState(newState);
   }
 
+  calculateTabWidthAndInkbarPosition(tabs, selectedIndex, defaultWidth) {
```

Comment:
We don't rely on the instance, it would be better to have this function as a module function.